src/pipecat/processors/aggregators/llm_response_universal.py

```diff
@@ -610,6 +619,8 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             await self.push_frame(frame, direction)
         elif isinstance(frame, LLMFullResponseStartFrame):
             await self._handle_llm_start(frame)
+        elif isinstance(frame, LLMTextFrame):
```

Comment:
Maybe worth a comment that this _must_ appear in this `if/elif` chain above `TextFrame` handling.